superset/assets/utils/common.js

```diff
@@ -39,8 +39,19 @@ export function getParamFromQuery(query, param) {
   return null;
 }
 
-export function getLink(baseUrl, params) {
-  return baseUrl + '?' + params.join('&');
+export function getQueryLink(query, callback) {
+  $.ajax({
+    type: 'POST',
+    url: '/kv/store/',
+    async: false,
+    data: {
+      baseUrl: 'superset/sqllab',
```

Comment:
I believe it should be stored in the backend only, flask already knows the url_for sql_lab